### DISCUSSION TITLE
Option to Make Login Screen Default to Overseerr Account

### DISCUSSION THREAD

**Discussion by 44vango** [sct/overseerr#3308] [Feature Requests]
Thanks so much for this great project!
I am only using local accounts and it would be helpful if there was an option in settings to have the login screen default to Overseerr account login instead of Plex account.
My users either don't notice the option under the **Sign In** button or forget and login fails.

![Screenshot 2023-02-01 at 21-39-09 Sign In - Overseerr](https://user-images.githubusercontent.com/124227455/216226010-4487e789-bc61-4282-af65-a2c6189f77f7.png)

**Comment by gangefors** [comment]
After installing Overseerr for the first time and adding an account this was the first setting I was looking for.
Please add an option for this, or at least make the local login form addressable with the URL with a param or something.

**Comment by mcmasterp** [comment]
Yes I need this for my users please

**Comment by frequencylost** [comment]
This please. For Plex servers that consist of managed users, Overseer pushing Plex Sign In front and centre is counter intuitive. 

All this needs to resolve user confusion is the Local Login to also have a Sign In button on the login page. So when Local Login checkbox is enabled, two Sign In Buttons will be visible. An upper Sign In button for Plex, and a lower Sign In button for local. 

When two buttons are seen, then users will actively think which one they need
